pkg/controller/plan/adapter/openstack/builder.go

```diff
@@ -216,19 +216,6 @@ func (r *Builder) ResolvePersistentVolumeClaimIdentifier(pvc *core.PersistentVol
 	return ""
 }
 
-// Build credential secret.
-func (r *Builder) Secret(_ ref.Ref, in, secret *core.Secret) (err error) {
```

Comment:
you need to have this function and just return for the builder API.

Reply:
yeah, already done